commits.md

```diff
@@ -0,0 +1,125 @@
+Table of Contents:
+1. [Commit Guidelines](#commit-guidelines)
+2. [Pull Request Guidelines](#pull-request-guidelines)
+
+
+
+# Commit Guidelines
+## How to Commit
+#### Separate by Concept
+It should be possible when skimming the commit summaries (`git log --oneline`)
+to have a good sense of what's being changed. For this reason and others
+(cherry-picking, organization, etc.), this means that the substance
+of each commit should be something you can summarize in a line. If you've made
+changes to a module to provide support for some new file format, for example,
+you should try to keep those changes in separate commits from completely
+unrelated changes (for example, adding testing for an already-supported
+file format). You may even want to keep changes that different on distinct
+branches and make distinct pull requests for them.
+
+#### Separate Moving and Modifying Files
```

Comment:
I don't think GitHub has the describe shortcomings actually. I happened to rename and edit a file at some point, and the GitHub diff looks just fine, and the diff box header shows the rename: https://github.com/SeattleTestbed/repy_v2/pull/134/files#diff-08d644fdf23e49274b1b1fb57fb15bb1 .

(Also, I'm not sure about unconditionally separating moves and edits for the other reasons given. The interdependency of not-yet reviewed PRs can be a drag.)